src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsStartupAttribute.cs

```diff
@@ -5,10 +5,20 @@
 
 namespace Microsoft.Azure.WebJobs.Hosting
 {
+    /// <summary>
+    /// Attribute used to declare <see cref="IWebJobsStartup"/> Types that should be registered and invoked
+    /// as part of host startup.
+    /// </summary>
     [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = true)]
     public class WebJobsStartupAttribute : Attribute
     {
-        public WebJobsStartupAttribute(Type startupType)
+        /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        /// <param name="startupType">The Type of the <see cref="IWebJobsStartup"/> class to register.</param>
+        /// <param name="alias">The friendly name alias for the startup action. If null, the name will be
+        /// defaulted based on naming convention.</param>
+        public WebJobsStartupAttribute(Type startupType, string alias = null)
```

Comment:
Are we keeping the alias? Did we want this to be a description?

Reply:
Clarified comments that this is a human readable alias, used for logging, etc.